packages/pyolite-kernel/src/kernel.ts

```diff
@@ -288,8 +291,6 @@ export class PyoliteKernel extends BaseKernel implements IKernel {
    */
   private async _sendWorkerMessage(type: string, data: any): Promise<any> {
```

Comment:
We should maybe rename this so that it's clearer that it must be used for requests **only**, something like `_sendRequestMessageToWorker`

Reply:
Done!